src/functions-and-arrays.js

```diff
@@ -1,73 +1,431 @@
 // Iteration #1: Find the maximum
 
+var maxOfTwoNumbers = [5, 7];
+
+function findMax(array) {
+  var max = -Infinity;
```

Comment:
Not really a need to start from -Infinity but ok I guess hehe. Like 0 would have been fine.